cmd/soroban-cli/src/commands/keys/fund.rs

```diff
@@ -10,6 +21,26 @@ pub enum Error {
     Address(#[from] address::Error),
     #[error(transparent)]
     Network(#[from] network::Error),
+    #[error(transparent)]
+    Secret(#[from] secret::Error),
+    #[error(transparent)]
+    Parsing(#[from] parsing::Error),
+    #[error(transparent)]
+    Rpc(#[from] rpc::Error),
+    #[error(transparent)]
+    Xdr(#[from] xdr::Error),
+    #[error(transparent)]
+    Clap(#[from] clap::Error),
+    #[error("RPC URL is missing in the network configuration")]
+    MissingRpcUrl,
+    #[error("Network passphrase is missing in the network configuration")]
+    MissingNetworkPassphrase,
+    #[error("Asset contract could not be deployed")]
+    AssetContractError,
+    #[error("Failed to transfer funds: {0}")]
+    FundTransferError(String),
```

Comment:
```suggestion
    FundTransferError(invoke::Error),
```
How does this create an recursive dependency? Does `invoke` use fund?

Reply:
Config::Error contains Keys::Error which is using Invoke::Error which also contains Config::Error

Reply:
Ah I see. This is definitely fine for now.